utils/build/docker/nodejs/express4-typescript.Dockerfile

```diff
@@ -23,8 +23,9 @@ ENV PGPORT=5433
 ENV DD_DATA_STREAMS_ENABLED=true
 
 # docker startup
-RUN printf '#!/bin/bash\nnode dist/app.js' > app.sh
-RUN chmod +x app.sh
+COPY utils/build/docker/nodejs/app.sh app.sh
+RUN printf '#!/bin/bash\nnode dist/app.js' > server.sh
+RUN chmod +x server.sh
 CMD ./app.sh
```

Comment:
Sometimes I confuse between app.js, app.sh, server.js and server.sh

Reply:
Not sure what action item is asked for here.